app/controllers/events/view/index.js

```diff
@@ -1,51 +1,76 @@
 import Controller from '@ember/controller';
+import { computed, action } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
 
-export default Controller.extend({
-  sponsorsColumns: [
-    {
-      propertyName   : 'logo-url',
-      template       : 'components/ui-table/cell/cell-image',
-      title          : 'Logo',
-      disableSorting : true
-    },
-    {
-      propertyName : 'name',
-      title        : 'Name'
-    },
-    {
-      propertyName   : 'type',
-      template       : 'components/ui-table/cell/cell-sponsor-sanitize',
-      title          : 'Type',
-      disableSorting : true
-    },
-    {
-      propertyName : 'level',
-      template     : 'components/ui-table/cell/cell-sponsor-sanitize',
-      title        : 'Level'
-    },
-    {
-      title          : 'Options',
-      template       : 'components/ui-table/cell/cell-sponsor-options',
-      disableSorting : true
-    }
-  ],
-  actions: {
-    deleteSponsor(sponsor) {
-      this.set('isLoading', true);
-      sponsor.destroyRecord()
-        .then(() => {
-          this.notify.success(this.l10n.t('Sponsor has been deleted successfully.'));
-          this.get('model.sponsors').removeObject(sponsor);
-        })
-        .catch(() => {
-          this.notify.error(this.l10n.t('An unexpected error has occurred.'));
-        })
-        .finally(() => {
-          this.set('isLoading', false);
-        });
-    },
-    editSponsor() {
-      this.transitionToRoute('events.view.edit.sponsors');
-    }
+export default class extends Controller.extend(EmberTableControllerMixin) {
+  per_page = 100;
```

Comment:
why?

Reply:
It was done in the same way for all `Admin Area`

Reply:
No, that was because logs had to be  intentionally sorted as we need to see 100 of the logs, not valid here. 
Also, `same way for all Admin Area`
This is simply not true, for admin events, admin users, admin sessions. They comprise the bulk of the admin area.
@kushthedude

Reply:
Removed it